assets/backstop_data/engine_scripts/casper/loadCookies.js

```diff
@@ -0,0 +1,15 @@
+var fs = require('fs');
+
+module.exports = function (casper, scenario) {
+  var cookies = [];
+  var cookiePath = scenario.cookiePath;
+
+  // READ COOKIES FROM FILE IF EXISTS
```

Comment:
I'm not sure this comment is require

Reply:
Oh all of these are not mine 😸

Reply:
@fabientownsend maybe you should raise a PR on the BackstopJS repo 😂